web/sites/default/config/core.entity_form_display.group.base_group.default.yml

```diff
@@ -821,14 +811,10 @@ content:
       placeholder: ''
     third_party_settings: {  }
   field_service_area:
```

Comment:
We need to add permissions for the Authenticated user role to edit the service area field, so anyone (e.g. group admins) who can edit the group entity can change the service area.